paddle/fluid/framework/selected_rows.h

```diff
@@ -118,7 +118,55 @@ class SelectedRows {
    *
    * @return index of the key.
    */
-  int64_t AutoGrownIndex(int64_t key, bool auto_grown, bool is_test = false);
+  int64_t AutoGrownIndex(int64_t key, bool auto_grown, bool is_test = false) {
```

Comment:
accelerate AutoGrownIndex by inline it

Reply:
done

Reply:
I don't change the implementation and usage now, just add a new inline function to access unordered map "rows_content -> rows_index".